main.typ

```diff
@@ -40,6 +40,20 @@
 
 = Typical Open Source Development
 
+Open source software development is a collaborative effort between developers and users. The users of the software are able to contribute to the development of the software by reporting bugs, suggesting features, and even contributing code. The developers of the software are able to use the feedback from the users to improve the software. Open source software development differs from traditional software development in two major ways. The first being that the users are able to contribute to the development of the software. The second being that almost all collaboration between developers happens asynchronously, through the internet.
```

Comment:
Is this just from your mind or from a paper you read? Anywhere that you learned from something make sure to cite it and then break it up with your own thoughts.

Reply:
This is from personal experience, but I can try to find some sources if you think that would be better.

Reply:
if you could i think it would make our argument stronger